packages/libgee.rb

```diff
@@ -1,6 +1,6 @@
-require 'package'
+require 'buildsystems/autotools'
 
-class Libgee < Package
+class Libgee < Autotools
   description 'Libgee is an utility library providing GObject-based interfaces and classes for commonly used data structures.'
   homepage 'https://wiki.gnome.org/Projects/Libgee'
   version '0.20.4'
```

Comment:
Update to [0.20.6](https://gitlab.gnome.org/GNOME/libgee/-/tags/0.20.6)?

Reply:
Doesn't build.